subprojects/gradle/module-dependencies-graph/src/gradleTest/kotlin/com/avito/module/dependencies/DependenciesGraphTestProject.kt

```diff
@@ -0,0 +1,73 @@
+package com.avito.module.dependencies
+
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.dependencies.GradleDependency
+import com.avito.test.gradle.module.AndroidAppModule
+import com.avito.test.gradle.module.AndroidLibModule
+import com.avito.test.gradle.module.KotlinModule
+import com.avito.test.gradle.plugin.plugins
+import java.io.File
+
+object DependenciesGraphTestProject {
```

Comment:
I'm scared of shared test fixtures for Gradle tests.
They tend to mix not related cases and become like God's objects.

Reply:
that's not a test fixture. It has a small scope that can't be wider

Reply:
By "test fixture" I mean not a technical solution but the idea of shared prepared data for tests. This is only a pattern.

Ok, I see the point about scope. Let it be.